lib/src/utils/constants.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
+abstract class Constants {
+  static const widthHeightOfDice = 100.0;
+  static const diceAmount = 6;
+  static const textHeight1_4 = 1.4;
+  static const fontWeight400 = FontWeight.w400;
```

Comment:
This const declaration is not necessary because FontWeight is already a constant.

Reply:
Done